plugins/main/bridge/bridge.go

```diff
@@ -124,6 +138,59 @@ func loadNetConf(bytes []byte, envArgs string) (*NetConf, string, error) {
 	return n, n.CNIVersion, nil
 }
 
+// This method is copied from https://github.com/k8snetworkplumbingwg/ovs-cni/blob/main/pkg/plugin/plugin.go
```

Comment:
Minor change. It may be useful to pin this link to a release version instead of following main.

Reply:
good idea
thanks a lot